osu.Game/Skinning/Editor/SkinComponentToolbox.cs

```diff
@@ -23,6 +23,8 @@ namespace osu.Game.Skinning.Editor
 {
     public class SkinComponentToolbox : ScrollingToolboxGroup
     {
+        public const float WIDTH = 200;
```

Comment:
Bit of a weird change to extract this public constant, because it doesn't seem to actually be used anywhere else in this diff - accidental, I'll presume?

Reply:
Was used in a previous iteration. I was trying to make the scaling on the skin editor actually take up the full available size, rather than "best effort" relative, but it turned out to be a bit more involved so will try again at a later point.

Reply:
That said, it's probably fine to leave this exposed?

Reply:
Sure. Was just mostly curious if anything.